cpp/src/io/json/nested_json.hpp

```diff
@@ -16,10 +16,17 @@
 
 #pragma once
 
+#include <cudf/io/types.hpp>
+#include <cudf/utilities/default_stream.hpp>
+
```

Comment:
Remove blank line separating `cudf` includes. Sort the ones that remain in this section (clang-format will do it).
```suggestion
```

Reply:
Thanks for catching these 👍  Done.